packages/trader/src/index.ts

```diff
@@ -1,33 +1,136 @@
 import { config } from "dotenv";
```

Comment:
this is the bots main entry point. it is structure very similarly to other bot's `index.js` files. The main difference is the configs come from the `TraderConfig` object, rather than from within this `index` file. I think this is a great pattern as it makes things cleaner with a more well defined separation of concerns.